### PR TITLE
Add removeMember method

### PR DESCRIPTION
**Pull request by mii-chan**
Add a method that removes a KiiUser from their KiiGroup

### CODE REVIEW

**Comment by fkmhrk** [comment]
Please add Unit tests to TestGroupApi.java

**Comment by mii-chan** [comment]
Revise Unit tests in TestGroupApi.java

File: java/src/test/java/jp/fkmsoft/libs/kiilib/apis/app/TestGroupApi.java [hunk 1]
```diff
@@ -162,7 +162,27 @@ public void onError(Exception e) {
     }
 
     @Test
-    public void test_0500_changeName() {
+    public void test_0500_removeMember() {
+        mClient.addToSendJson(204, null, "");
+
+        KiiGroup group = new KiiGroup("group2345");
+
```
Comment (fkmhrk) [inline java/src/test/java/jp/fkmsoft/libs/kiilib/apis/app/TestGroupApi.java]: Please add initial members

File: java/src/test/java/jp/fkmsoft/libs/kiilib/apis/app/TestGroupApi.java [hunk 2]
```diff
@@ -162,7 +162,27 @@ public void onError(Exception e) {
...
+        KiiUser user = new KiiUser("user2345");
+        mAppApi.groupAPI().removeMember(group, user, new GroupAPI.GroupCallback<KiiGroup>() {
+            @Override
+            public void onSuccess(KiiGroup group) {
+                Assert.assertNotNull(group);
+            }
```
Comment (fkmhrk) [inline java/src/test/java/jp/fkmsoft/libs/kiilib/apis/app/TestGroupApi.java]: Please add an assertion : member is removed from KiiGroup object.

**Comment by mii-chan** [comment]
Add an initial member and an assertion

**Pull request merged**